gnosis/safe-react
# Update custom Safe Apps link

## What it solves
Resolves #2891

## How this PR fixes it
The [old custom Safe Apps link](https://docs.gnosis.io/safe/docs/sdks_safe_apps/) was updated to the [new one](https://docs.gnosis-safe.io/build/sdks/safe-apps) and changed to a constant, outside of the component.

## How to test it
Open the Apps panel, click the "Add custom app" button and the "Learn more about building Safe Apps." link. It should open our new GitBook docs.
